### DISCUSSION TITLE
Why no adapter will work in my 2016 Silverado LTZ

### DISCUSSION THREAD

**Discussion by bsenseman** [ludwig-v/wireless-carplay-dongle-reverse-engineering#109] [Q&A] [answered]
Since the various vendors; CPlay2Air & Ottocast are no help, anybody here have a thought on why neither adapter works in my 2016 Silverado?  The CPlay one worked (yes past tense) in my 16 GMC, but not the Chevy; then it died, no power to a plugged in USB stick to flash...  Same results with a newer Ottocast U2-NOW, but its still functional.  Confuzzled on why it works in one GM 2016 but not the other, both have Navigation too.

**Comment by jsm174** [comment, marked as answer]
I had the same thing in my Silverado and in my wife's Tahoe.

I was able to get it to work after resetting the radio. 

![IMG_7679](https://user-images.githubusercontent.com/1197137/132906426-2f9a9ac0-3050-42bd-b306-fd629b5a9907.jpeg)
![IMG_7680](https://user-images.githubusercontent.com/1197137/132906427-b89970af-5da2-49f3-bc7c-ecefc09af8a0.jpeg)

**Reply by bsenseman** [reply]
Genius!  It worked.  Not sure why I did not have to do this with my GMC, but it was new the Chevy was used, so I guess maybe too many devices for the radio.  But it INSTANTLY showed Carplay!!

**Reply by jsm174** [reply]
Awesome. Glad it worked!
